assets/js/components/SaptuneDetails/SaptuneDetails.test.jsx

```diff
@@ -102,4 +102,36 @@ describe('SaptuneDetails', () => {
       staging.solutions_ids
     );
   });
+
+  it('should display dashes for enabled and applied solutions when solution is null', () => {
+    const solution = null;
+
+    const {
+      configured_version: configuredVersion,
+      package_version: packageVersion,
+      services,
+      staging,
+      tuning_state: tuningState,
+    } = saptuneStatusFactory.build();
+    const { hostname, id: hostID } = hostFactory.build();
+    renderWithRouter(
+      <SaptuneDetails
+        appliedSolution={solution}
+        enabledSolution={solution}
```

Comment:
Let's add `enabledNotes` and `appliedNotes` with empty lists, so we check a dash is rendered.
And a "disabled" staging (empty solution and notes)